src/py42/clients/alerts.py

```diff
@@ -25,20 +25,22 @@ def search(self, query):
                 See `userguides.searches` to learn more about how to construct a query.
 
         Returns:
-            :class:`py42.sdk.response`: A response containing the alerts that match the given query.
+            :class:`py42.sdk.response`: A response containing the alerts that match the given
+            query.
         """
         query = str(query)
         uri = self._uri_prefix.format(u"query-alerts")
         return self._session.post(uri, data=query)
 
     def get_query_details(self, alert_ids, tenant_id=None):
         """Gets the details for a file event query that when passed into a search would result in
-        events that could be responsible for the alerts with the given alert IDs.
+        events that could be responsible for the alerts with the given IDs.
```

Comment:
In the Demisto integration, they used this method to create the query somehow. Why do you feel it is misnamed? Because it sounds like it could be referring to an alert query?

Maybe something like `get_related_file_event_query()`?
I'm not sure I like `get_alert_detauls()` because it sounds like that function is what I need to call to get the Case ID and stuff

Reply:
@alanag13 most likely has some thoughts on this

Reply:
Ok, I see that the console UI does have a "Investigate in Forensic Search" button on the Alert detail page, that must transform the result into a FFS query. But that seems like a secondary use case from the alert detail results, the primary use case is displaying the alert events summary. 

Since it's not clear _how_ one would transform the results of `query-details` into an FFS query, shouldn't this method be `get_alert_details()` and then we have another method for actually doing the transformation from alert_details > FileEventQuery?

Reply:
I see that now, but when I first made this, I'm pretty sure it didn't.

Look at the `get_by_id` method... That calls an API named "details" off of Alerts... That is the true alert details, this API is for getting the file event query details, but since the UI must need both now, it must just use this one to save an API call.

Reply:
You can get all the same information using this method and `get_case_by_id`, so the secondary use case is all that is really needed anyway from this function.

Reply:
but there's no get_case_by_id method on the AlertClient, are you meaning the one on DepartingEmployeeClient? AlertIds are different than DE cases.

Reply:
Oh whoops I totally mixed those up again

Reply:
OK, I'll rename it

Reply:
I agree with this renaming